Release version 0.16.0

tijmenb opened the pull request:
Add `Smartdown::Api::Flow#content_id` (https://github.com/alphagov/smartdown/pull/147)

tijmenb committed:
Release version 0.16.0

Add `Smartdown::Api::Flow#content_id`

KushalP commented on CHANGELOG.md:
```diff
@@ -1,3 +1,5 @@
+## 0.16.1
```
Why are you incrementing both the minor version and the patch version here? I'm wondering if this should be `0.16.0` ?

tijmenb replied:
Heh, yeah. Thanks.

tijmenb committed:
Specify ruby-version to run tests against

Without this .ruby-version, the jenkins tests fail and the gem cannot
be released.

2.2.3 is the one used by smart-answers currently.

alext commented:
For gems, it's better to pin to a minor version (i.e. 2.2) rather than a specific version, that way it'll always be build against the latest 2.2.x. (We create rbenv aliases in all environments so that 2.2 points at the latest 2.2.x)

tijmenb commented:
@alext Thanks, didn't know that. Updated.
